docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,54 @@
+*******************************************************
+Staying Under /home/<username> Quota
+*******************************************************
+It is vital to stay under your /home/<username> quota to prevent issues in using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your /home/<username> directory.
```

Comment:
i prefer "stay under your quota at /home/<username>", but that is a "salt to taste" issue.

Reply:
I switched to your title because I didn't like including <username> in my title but wanted to be specific to what the directory was in the RTD.

Reply:
The title seems odd. Feel free to disagree. In case you consider modifying, here are a few examples:

- Tips for Managing User's /home/<username> Quota
- Best Practices for Data Storage and Management
- Storage Quota and Managing Large Files on Our HPC Platform

Reply:
I agree with @visionjo that we can slightly modify the title to indicate that it's for $HOME, so something like: 
Managing $HOME Quota